src/aurelia-i18n.js

```diff
@@ -51,16 +51,17 @@ function configure(frameworkConfig, cb): Promise<void> {
 
   // check whether Intl is available, otherwise load the polyfill
   if (window.Intl === undefined) {
+    console.info('Intl API is not available. Trying to load the polyfill.');  // eslint-disable-line no-console
     let loader = frameworkConfig.container.get(Loader);
 
     return loader.normalize('aurelia-i18n').then((i18nName) => {
       return loader.normalize('intl', i18nName).then((intlName) => {
         return loader.loadModule(intlName).then((poly) => {
           window.Intl = poly;
           return registerI18N(frameworkConfig, cb);
-        });
-      });
-    });
+        }, () => console.warn('Failed to load the Intl polyfill.'));  // eslint-disable-line no-console
+      }, () => console.warn('Failed to normalize intl module while loading the Intl polyfill.'));  // eslint-disable-line no-console
+    }, () => console.warn('Failed to normalize aureli-i18n while loading the Intl polyfill.'));  // eslint-disable-line no-console
```

Comment:
We've got some duplicated error message strings. Can you factor those out into a const and reference that? Also, please switch this to using the logger. Thanks!